docs/how-to-contribute.md

```diff
@@ -1,3 +1,109 @@
+* **Perform a [cursory search](https://github.com/search?q=+is%3Aissue+user%3Aopeneuropa)** to see if the problem has already been reported. If it has **and the issue is still open**, add a comment to the existing issue instead of opening a new one.
+
+#### How Do I Submit A (Good) Bug Report?
+
+Bugs are tracked as [GitHub issues](https://guides.github.com/features/issues/). After you've determined [which repository](#before-getting-started) your bug is related to, create an issue on that repository and provide the following information by filling in [the template](issue-template.md).
+
+Explain the problem and include additional details to help maintainers reproduce the problem:
+
+* **Use a clear and descriptive title** for the issue to identify the problem.
+* **Describe the exact steps which reproduce the problem** in as many details as possible.
+* **Describe the behavior you observed after following the steps** and point out what exactly is the problem with that behavior.
+* **Explain which behavior you expected to see instead and why.**
+* **Include screenshots** if they help understand the steps or the result.
+
+Provide more context by answering these questions:
+
+* **Did the problem start happening recently** (e.g. after updating to a new version of the component) or was this always a problem?
+* If the problem started happening recently, **can you reproduce the problem in an older version of the component?** What's the most recent version in which the problem doesn't happen?
+* **Can you reliably reproduce the issue?** If not, provide details about how often the problem happens and under which conditions it normally happens.
+
+Include details about your configuration and environment:
+
+* **Which version of the component are you using?**
+* **What's the name and version of the browser you are using**?
+* **Which other components or modules do you have installed?**
+
+### Feature requests
+
+This section guides you through submitting a feature request for OpenEuropa, including completely new features and minor improvements to existing functionality.
+
+Before creating a feature request, please check [this list](#before-submitting-a-feature-request) as you might find out that you don't need to create one. When you are creating a feature request, please [include as many details as possible](#how-do-i-submit-a-good-enhancement-suggestion). Fill in [the template](issue-template.md), including the steps that you imagine you would take if the feature you're requesting existed.
+
+#### Before Submitting A Feature Request
+
+* **Check if there's already [a component](openeuropa-components.md) which provides that enhancement.**
+* **Determine [which repository the feature should be suggested in](#before-getting-started).**
+* **Perform a [cursory search](https://github.com/search?q=+is%3Aissue+user%3Aopeneuropa)** to see if the feature has already been suggested. If it has, add a comment to the existing issue instead of opening a new one.
+
```

Comment:
Add  ** Check the project roadmap to understand if the feature is already being planned for future releases associated dates.